lib/coinbase.js

```diff
@@ -22,16 +22,17 @@ export default class Coinbase {
     return Promise.all(
       this.cryptoCurrencies.map(cryptoCurrency =>
         this.getPrice({ action, currencyPair: `${cryptoCurrency}-${this.localCurrency}` })
-          .then((price) => {
+          .then((value) => {
             const body = {
               amount: 1,
               currency: cryptoCurrency,
               localCurrency: this.localCurrency,
-              price,
+              value,
               metricName: `${action}Price`,
+              namespace: `Coinbase`,
```

Comment:
Exactly, and you are mentioning `coinbase` as main namespace. So I got confused.